crates/typst/src/eval/methods.rs

```diff
@@ -172,7 +172,13 @@ pub fn call(
             "keys" => dict.keys().into_value(),
             "values" => dict.values().into_value(),
             "pairs" => dict.pairs().into_value(),
-            _ => return missing(),
+            _ => {
+                return if matches!(dict.at(method, None), Ok(Value::Func(_))) {
+                    Err(missing_method(name, method)).hint(eco_format!("to call the function stored in the dictionary, surround the field access with parentheses")).at(span)
```

Comment:
this line is really long, I'm surprised that rustfmt doesn't do anything here ...

Reply:
Yeah, my guess is that it doesn't do line breaks because of the long string. So I let it format with a smaller string, and that seemed to do the trick